lib/forms/pegasus_form_validation.rb

```diff
@@ -4,6 +4,7 @@
 require_relative 'pegasus_form_errors'
 require_relative '../cdo/regexp'
 require_relative '../cdo/geocoder'
+require_relative '../../pegasus/forms/hoc_signup_2017'
```

Comment:
Actually, I rescind the good call. The test is agnostic to the class, it is simply referencing the string `HocSignup2017`. Though I'm going to keep things as-is, I do think the validation is missing the require, and my tests are exposing this.